README.md

```diff
@@ -393,6 +393,8 @@ For more information about the governance of the Node.js project, see
 * [jasongin](https://github.com/jasongin) -
 **Jason Ginchereau** &lt;jasongin@microsoft.com&gt;
 * [jbergstroem](https://github.com/jbergstroem) -
+**John-David Dalton** &lt;john.david.dalton@gmail.com&gt;
```

Comment:
The lines seem in the wrong order)

![1](https://user-images.githubusercontent.com/10393198/40561964-e608abe0-6067-11e8-8506-83034d60e2f8.png)